core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -100,7 +100,7 @@ public static void dropTableData(FileIO io, TableMetadata metadata) {
 
     LOG.info("Manifests to delete: {}", Joiner.on(", ").join(manifestsToDelete));
 
-    // run all of the deletes
+    // run all the deletes
```

Comment:
Probably fine as is?

Reply:
IntelliJ trying to be smart at syntax so I let it fix but I can revert!